tests/cpp/test_gpu3.cpp

```diff
@@ -8770,6 +8771,56 @@ TEST_F(NVFuserTest, Issue2685Repro) {
   testValidate(&fusion_copy, outputs, inputs, __LINE__, __FILE__);
 }
 
+// Check that extents are properly replaced by replaceSymbolicSizes lowering
+// pass
+TEST_F(NVFuserTest, ReplaceSymbolicSizes) {
```

Comment:
Added this test which matches the behavior described in the comments of `replaceSymbolicSizes`.